acceptance/setup/aio/pre-suite/010_Install.rb

```diff
@@ -44,3 +46,25 @@
     end
   end
 end
+
+step 'Install build dependencies on master' do
+
+ MASTER_PACKAGES = {
+    :redhat => [
+      'git',
+      'java-1.8.0-openjdk-devel',
+    ],
+  }
+  install_packages_on(master, MASTER_PACKAGES, :check_if_exists => true)
+end
+
+step 'Confirm SSH Forwarding Enabled on master' do
```

Comment:
The git clone of pcp-broker via git (rather than http) depends on it. The previous code (before this PR) was doing the git clone over http and got away without having the github key set properly.